### DISCUSSION TITLE
Pie Metric visual inconsistency using dynamic height

### DISCUSSION THREAD

**Discussion by KasparRosin** [laravel/nova-issues#6233] [Ideas & Feature Requests] [answered]
- Laravel Version: latest
- Nova Version: latest

### Description:
When using dynamic height alongside the pie metric, the colored dots disappear from the beginning. Would be visually nice if they could continue showing different colors.

### Detailed steps to reproduce the issue on a fresh Nova installation:
![image](https://github.com/laravel/nova-issues/assets/33309407/e589c144-a6cb-4c02-8d1a-2e48f3f15f57)

**Comment by crynobone** [comment, marked as answer]
Duplicate of #6183
